cmd/odf/maintenance/start.go

```diff
@@ -0,0 +1,22 @@
+package maintenance
+
+import (
+	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
+	"github.com/rook/kubectl-rook-ceph/pkg/maintenance"
+	"github.com/spf13/cobra"
+)
+
+var startMaintenanceCmd = &cobra.Command{
+	Use:     "start",
+	Short:   "Start a maintenance deployment with an optional alternative ceph container image",
+	Args:    cobra.ExactArgs(1),
+	Example: " odf maintenance start <deployment_name>",
+	Run: func(cmd *cobra.Command, args []string) {
+		alternateImage := cmd.Flag("alternate-image").Value.String()
+		maintenance.StartMaintenance(cmd.Context(), root.ClientSets.Kube, root.OperatorNamespace, args[0], alternateImage)
+	},
+}
+
+func init() {
```

Comment:
why there is a separate init function, if the alternate image is already passed in the function
or it should be only called if user has passed it

Reply:
So that the cobra knows it's flag and initializes the flag with the command